pages/windows/prompt.md

```diff
@@ -0,0 +1,24 @@
+# prompt
+
+> Change the default DOS style prompt in a command window.
+> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/prompt>.
+
+- Reset the prompt to default 'C:\>':
+
+`prompt`
+
+- Change the prompt line:
+
+`prompt {{special_codes}}`
```

Comment:
```suggestion
- Set a specific prompt:

`prompt {{prompt}}`
```

Optional fix: `a specific` may sound better.
Optional fix: placeholder can be simplified.